### PR TITLE
P#51853 Add PHP 8 to unit tests

### PR DESCRIPTION
**Pull request by LongTrong-exe**

### CODE REVIEW

**Comment by LongTrong-exe** [comment]
@jmaas-onoffice Needed phpunit 9. to support coverage so I updated it too

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.2','7.3', '7.4']
+        php-versions: ['7.3', '7.4', '8.0']
```
Comment (jmaas-onoffice) [inline .github/workflows/unit-tests.yml:19]: Our [official minimal PHP version](https://de.wordpress.org/plugins/onoffice-for-wp-websites/) for the plugin is 7.2. What do you think about having `['7.2', '7.4', '8.0']`?

Comment (LongTrong-exe) [reply .github/workflows/unit-tests.yml:19]: But phpunit 9 requires php 7.3 or higher

Comment (LongTrong-exe) [reply .github/workflows/unit-tests.yml:19]: @jmaas-onoffice I think we can skip run unittest at php7.2

Comment (jmaas-onoffice) [reply .github/workflows/unit-tests.yml]: Then I think you need to revert 6fc5136a46aba4b15426477f3017cacf0f81d497. We can just update the minimum PHP version and make a new major release.

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.2','7.3', '7.4']
+        php-versions: ['7.2', '7.4', '8.0']
```
Comment (jmaas-onoffice) [inline .github/workflows/unit-tests.yml:19]: I think it's better to revert 6fc5136a46aba4b15426477f3017cacf0f81d497. We will update the minimum PHP version.

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — DISMISSED** [review]

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — DISMISSED** [review]

**Review by jmaas-onoffice — APPROVED** [review]

**Pull request merged**